schemas/com.snowplowanalytics.snowplow.enrichments/pii_pseudonymizer_config/1-0-0

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Path is wrong. It's missing `jsonschema` in `pii_pseudonymizer_config/1-0-0`.

Reply:
Oh I see. Not on that line. In the actual path. Ok I see it.